### PR TITLE
homework_4.03_android_components

### PR DESCRIPTION
**Pull request by AliakseiShvants**
Created simple fragment, added service and broadcast receiver

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -8,7 +9,17 @@
...
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        tools:ignore="GoogleAppIndexingWarning">
+
+        <service android:name=".CounterService" />
+
+        <receiver android:name=".CounterBroadcastReceiver">
+            <intent-filter>
+                <action android:name="com.shvants.CUSTOM_ACION" />
```
Comment (ajaxspace) [inline app/src/main/AndroidManifest.xml]: Read about https://developer.android.com/studio/build/manifest-build-variables.

File: app/src/main/java/com/shvants/runninglife/MainActivity.java [hunk 2]
```diff
@@ -1,13 +1,35 @@
...
 import android.os.Bundle;
+import android.os.IBinder;
 import android.support.v7.app.AppCompatActivity;
 import android.view.View;
-import android.widget.Button;
 
 
 public class MainActivity extends AppCompatActivity {
 
+    public static final String RESULT = "RESULT";
```
Comment (ajaxspace) [inline app/src/main/java/com/shvants/runninglife/MainActivity.java]: There is already such field in CounterBroadcastReceiver

File: app/src/main/java/com/shvants/runninglife/MainActivity.java [hunk 3]
```diff
@@ -26,5 +47,63 @@ public void onClick(final View v) {
...
+            public void onClick(final View v) {
+                start = System.currentTimeMillis();
+                startService(serviceIntent);
+                bindService(serviceIntent, connection, BIND_AUTO_CREATE);
+            }
+        });
+
+        findViewById(R.id.main_stop_service_btn).setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(final View v) {
```
Comment (ajaxspace) [inline app/src/main/java/com/shvants/runninglife/MainActivity.java:67]: Calling this logic if service is not started will lead to crash.

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (AliakseiShvants) [reply app/src/main/AndroidManifest.xml]: applied 'applicationId' https://github.com/AliakseiShvants/RunningLife/compare/01842e8f4dd29c265bdc458cc8a8a71b6eae8310...58e65bc2b5d572145ec44afbcbc683ee2b932035

File: app/src/main/java/com/shvants/runninglife/MainActivity.java (hunk 2 above)
Comment (AliakseiShvants) [reply app/src/main/java/com/shvants/runninglife/MainActivity.java]: moved this constant in 'Const' class https://github.com/AliakseiShvants/RunningLife/compare/01842e8f4dd29c265bdc458cc8a8a71b6eae8310...58e65bc2b5d572145ec44afbcbc683ee2b932035

File: app/src/main/java/com/shvants/runninglife/MainActivity.java (hunk 3 above)
Comment (AliakseiShvants) [reply app/src/main/java/com/shvants/runninglife/MainActivity.java:67]: @ajaxspace made stop button not clickable until start button is pressed https://github.com/AliakseiShvants/RunningLife/compare/01842e8f4dd29c265bdc458cc8a8a71b6eae8310...58e65bc2b5d572145ec44afbcbc683ee2b932035

**Pull request closed without merging**